topo/node/cptx/cptx.go

```diff
@@ -140,6 +140,26 @@ func (n *Node) GenerateSelfSigned(ctx context.Context) error {
 		}
 	}
 
+	cfgs := []string{
+		"set system services extension-service request-response grpc ssl hot-reloading",
+		"set system services extension-service request-response grpc ssl use-pki",
+		"set openconfig-system:system openconfig-system-grpc:grpc-servers grpc-server grpc-server config services GNMI",
+		"set openconfig-system:system openconfig-system-grpc:grpc-servers grpc-server grpc-server config enable true",
+		"set openconfig-system:system openconfig-system-grpc:grpc-servers grpc-server grpc-server config port 32767",
+		"set openconfig-system:system openconfig-system-grpc:grpc-servers grpc-server grpc-server config transport-security true",
+		"set openconfig-system:system openconfig-system-grpc:grpc-servers grpc-server grpc-server config certificate-id grpc-server-cert",
+		"set openconfig-system:system openconfig-system-grpc:grpc-servers grpc-server grpc-server config listen-addresses 0.0.0.0",
+		"commit",
+	}
+	resp, err := n.cliConn.SendConfigs(cfgs)
```

Comment:
why are you sending this now as cli vs. in the initial config?

Reply:
Having this config part of the default config causes a race where gRPC service is started before the certer interface is invoked leading to failure. This should be configured after self-signed certs are configured.